docs/03-TechnicalInfrastructure.md

```diff
@@ -15,19 +15,20 @@ The root directory of the imaging part of a LORIS instance is typically
         |__ bin
         |   |__ mri
         |__ data
-            |__ trashbin
-            |__ tarchive
-            |__ pic
-            |__ jiv
-            |__ logs
             |__ assembly
             |__ batch_output
-            |__ pipelines*
+            |__ jiv
+            |__ logs
+                |__ DTIPrep_pipeline*
+                |__ DTIPrep_register*
+            |__ pic
+            |__ tarchive
+            |__ trashbin
             |__ protocols*
```

Comment:
missing `pipelines` in the tree

Reply:
okay re-added it. But I removed it intentionally because the code never makes one, so I am not sure where it comes from?